src/Voron/Schema/Updates/From23.cs

```diff
@@ -20,13 +20,9 @@ public bool Update(int currentVersion, StorageEnvironmentOptions options, Header
                 // to recover some disk space, and everything will still functions fine with this
             }
         }
-        
-        headerAccessor.Modify((ref FileHeader header) =>
-        {
-            // We are moving from a legacy system with no journals 
-            // to having a dedicated journal id for each env
-            header.JournalId = Guid.NewGuid();
-        });
+
+        Span<MetadataFile> metadata = stackalloc MetadataFile[1];
+        headerAccessor.MetadataAccessor.FillMetadata(ref metadata[0]);
```

Comment:
This doesn't actually do something, does it?
I assume that you are relying that we are later calling persist explicitly?
Why not persist _here_ ?

Reply:
But you are just modifying a value on the stack?